automapper/src/main/java/com/valqueries/automapper/ValqueriesCrudRepositoryBase.java

```diff
@@ -151,7 +171,7 @@ protected String getTableName() {
 		return getTableName(Clazz.of(modelType));
 	}
 
-	String getTableName(Clazz<?> modeltype) {
+	String getTableName(Clazz<? extends Object> modeltype) {
```

Comment:
Maybe this method could be pulled up to a more general layer

Reply:
6366fb0aad02a398c80e33626a7b2f8075c7cce0